src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -312,6 +314,57 @@ public WorkspaceSelector(@Nullable MCreatorApplication application, WorkspaceOpe
 		}
 	}
 
+	private JPopupMenu getRightClickMenu() {

```

Comment:
When right-clicking entries and the popup opens, current entry below cursor should be selected, otherwise it is quite bad experience since not the entry mouse clicked on is the one the popup relates to